src/_includes/docs/swift-package-manager/how-to-enable-disable.md

```diff
@@ -0,0 +1,62 @@
+## How to enable Swift Package Manager
+
+Flutter's Swift Package Manager feature is disabled by default.
+To enable it:
+
+1. Switch to Flutter's `master` channel:
```

Comment:
do we still use master channel or is it renamed to main?

Reply:
Both exist but `master` is still the "real" one.

Reply:
If `main` works, let's go with main.

Reply:
Turns out I was wrong, switched to `main`!